### ISSUE TITLE
Error on starting: Player.log contains no player data.

### ISSUE THREAD

**Issue by zyll71** [Manuel-777/MTG-Arena-Tool#1223]
After the most recent MTGA update (today) to Version 2021.3.0.2470 3470.854029, MTG Arena Tool 5.5.10 stops at the login with "Player.log contains no player data." on OpenSuSE Tumbleweed Linux 5.11.2-1-default x86-64.

Player.log appears at the usual location and is not empty.

```
user@Tulkas:~> ./bin/MTG-Arena-Tool-5.5.10.AppImage
21:41:53.312 › linux
^[[24~21:41:53.883 › Db path: /home/user/.config/MTG-Arena-Tool/application.db
21:41:53.893 › Local DB: Loading data...
21:41:53.907 › Db backup: /home/user/.config/MTG-Arena-Tool/application_backup.db
21:41:53.915 › Local DB: Loading data... complete.
(electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397
Checking for update
^[[24~Update for version 5.5.10 is not available (latest version: 5.5.10, downgrade is disallowed).
21:41:54.783 › Update not available
21:41:54.783 › {
  version: '5.5.10',
  files: [
    {
      url: 'MTG-Arena-Tool-5.5.10.AppImage',
      sha512: 'Kxm7TOh7OO9R1F+RM1WyCcdwDMUz0bU0tbrrZoBJT+mvhUILRdQ/1XTXzml2ofh+b0snMihf1yhyv2Y09MsyKg==',
      size: 109977663,
      blockMapSize: 114771
    }
  ],
  path: 'MTG-Arena-Tool-5.5.10.AppImage',
  sha512: 'Kxm7TOh7OO9R1F+RM1WyCcdwDMUz0bU0tbrrZoBJT+mvhUILRdQ/1XTXzml2ofh+b0snMihf1yhyv2Y09MsyKg==',
  releaseDate: '2021-03-10T14:11:06.710Z',
  releaseName: 'v5.5.10',
  releaseNotes: ''
}
checkForUpdatesAndNotify called, downloadPromise is null
21:41:56.276 › MAIN:  Initializing
21:41:56.280 › MAIN:  Updating settings
21:41:56.283 › IPC LOG: Downloading metadata en
21:41:56.562 › IPC LOG: Initial log parse: arenaId=undefined
21:41:56.562 › IPC LOG: Initial log parse: playerName=undefined
21:41:57.387 › IPC LOG: Database up to date (110), skipping download.
21:47:33.593 › IPC LOG: Initial log parse: arenaId=undefined
21:47:33.595 › IPC LOG: Initial log parse: playerName=undefined
21:47:33.779 › IPC LOG: Initial log parse: arenaId=undefined
21:47:33.780 › IPC LOG: Initial log parse: playerName=undefined
21:47:34.529 › IPC LOG: Initial log parse: arenaId=undefined
21:47:34.530 › IPC LOG: Initial log parse: playerName=undefined
...
...
...
Bye bye!
```

Maybe a change in the log structure the parser can't deal with?

**Comment by riQQ** [comment]
Yes, WotC removed data from the logs once again.

**Comment by Merovech** [comment]
I don't know if this is related so I'll leave it here (I can open a new ticket if necessary) but a related behavior is that the login button is unresponsive.  I get the hand cursor, but clicking does nothing.

**Comment by randydork** [comment]
I have both of the above issues, data fiile empty, cannot logiin or click continue offline.  I did wonder if it was an ussue with Norton blocking the app but I couldnt see any issue there.

**Comment by riQQ** [comment]
A preliminary fix was implemented by #1228 and released in v5.5.13.

**Comment by zyll71** [comment]
The big no longer appears after the update.
Thanks for fixing it.
